test/UnifiedRegex/bugFixRegression.js

```diff
@@ -501,13 +501,6 @@ exec(/(?:a*)?/, "");
 exec(/(?:a+)?/, "");
 exec(/(?:a||b)?/, "b");
 
-// WOOB1145588
-exec(/[\s-a-c]*/, " -abc");
-exec(/[\s\-a-c]*/, " -abc");
-exec(/[a-\s-b]*/, " -ab");
-exec(/[a\-\s\-b]*/, " -ab");
-exec(/[\s--c-!]*/, " -./0Abc!");

```

Comment:
I didn't notice any tests for out-of-order range endpoints like `[c-!]` in the new test. We should preserve the test (but ensure that it now throws an error).

Reply:
Added the test case to the regressions.